src/python/grpcio_tests/tests_aio/unit/metadata_test.py

```diff
@@ -105,13 +100,13 @@ async def _test_client_to_server(request, context):
     async def _test_server_to_client(request, context):
         assert _REQUEST == request
         await context.send_initial_metadata(
-            _INITIAL_METADATA_FROM_SERVER_TO_CLIENT)
+            tuple(_INITIAL_METADATA_FROM_SERVER_TO_CLIENT))
```

Comment:
Maybe we can perform the type conversion inside `send_initial_metadata`? It should better reflect the fact that the `Metadata` type is supported officially.